test/pantry-test.js

```diff
@@ -0,0 +1,105 @@
+import { expect } from 'chai';
+
+import Pantry from '../src/pantry.js';
+import Recipe from '../src/recipe.js';
+import User from '../src/user.js';
+import recipeData from '../src/data/recipes.js';
+import ingredientsData from '../src/data/ingredients.js';
+
+
+
+describe('Pantry', function() {
+
+  let recipe, userData, userData2, user1, user2, pantry1, pantry2
+
+  beforeEach(function() {
+
+    userData = {
+      "id": 1,
+      "name": "Saige O'Kon",
+      "pantry": [
+        {
+          "ingredient": 11477,
+          "amount": 1
+        },
+        {
+          "ingredient": 93820,
+          "amount": 1
+        },
+        {
+          "ingredient": 11297,
+          "amount": 3
+        }]
+      }
+      userData2 = {
+        "id": 2,
+        "name": "Ab",
+        "pantry": [
+          {
+            "ingredient": 1041009,
+            "amount": 2
+          },
+          {
+            "ingredient": 10018413,
+            "amount": 1
+          },
+          {
+            "ingredient": 2044,
+            "amount": 3
+          },
+          {
+            "ingredient": 10111529,
+            "amount": 0.5
+          },
+          {
+            "ingredient": 4053,
+            "amount": 1
+          },
+          {
+            "ingredient": 11477,
+            "amount": 1
+          }]
+
+
+    };
+
```

Comment:
This area can remove unnecessary white spaces and correct indentation.  It is a minor thing, but it would be useful to make these changes early so that we won't have to worry about it in a later walkthrough of code to make overall code clean and proper.

Reply:
Ok, I fixed the white spacing and indentation issues